LocalSettings.php

```diff
@@ -4571,7 +4571,7 @@ function extractGlobals() {
 	global $wgConf, $wgDBname, $wgAPIModules;
```

Comment:
global `$wgRevokePermissions` though would this not be better to overwrite ManageWiki's $wgEnableUploads instead? If it is done after the wgConf extract it should work I think though can't guarantee that.